### PR TITLE
Update productionizing.md

### PR DESCRIPTION
**Pull request by simonkurtz-MSFT**
## Purpose

Add link to PR for openai-priority-loadbalancer to offer a native Python approach via an injected httpx transport.

## Does this introduce a breaking change?

When developers merge from main and run the server, azd up, or azd deploy, will this produce an error?
If you're not sure, try it out on an old environment.

```
[ ] Yes
[x] No
```

## Does this require changes to learn.microsoft.com docs?

This repository is referenced by [this tutorial](https://learn.microsoft.com/azure/developer/python/get-started-app-chat-template)
which includes deployment, settings and usage instructions. If text or screenshot need to change in the tutorial,
check the box below and notify the tutorial author. A Microsoft employee can do this for you if you're an external contributor.

```
[ ] Yes
[x] No
```

## Type of change

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## Code quality checklist

See [CONTRIBUTING.md](https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/CONTRIBUTING.md#submit-pr) for more details.

- [ ] The current tests all pass (`python -m pytest`).
- [ ] I added tests that prove my fix is effective or that my feature works
- [ ] I ran `python -m pytest --cov` to verify 100% coverage of added lines
- [ ] I ran `python -m mypy` to check for type errors
- [ ] I either used the pre-commit hooks or ran `ruff` and `black` manually on my code.

### CODE REVIEW

**Comment by simonkurtz-MSFT** [comment]
Hi @pamelafox, here's the requested PR.

File: docs/productionizing.md [hunk 1]
```diff
@@ -24,9 +24,10 @@ If the maximum TPM isn't enough for your expected load, you have a few options:
 
 * Use a backoff mechanism to retry the request. This is helpful if you're running into a short-term quota due to bursts of activity but aren't over long-term quota. The [tenacity](https://tenacity.readthedocs.io/en/latest/) library is a good option for this, and this [pull request](https://github.com/Azure-Samples/azure-search-openai-demo/pull/500) shows how to apply it to this app.
 
-* If you are consistently going over the TPM, then consider implementing a load balancer between OpenAI instances. Most developers implement that using Azure API Management or container-based load balancers. For seamless integration instructions with this sample, please check:
+* If you are consistently going over the TPM, then consider implementing a load balancer between OpenAI instances. Most developers implement that using Azure API Management or container-based load balancers. A native Python approach that integrates with the OpenAI Python API Library is also possible. For seamless integration instructions with this sample, please check:
```
Comment (pamelafox) [inline docs/productionizing.md:27]:
```suggestion
* If you are consistently going over the TPM, then consider implementing a load balancer between OpenAI instances. Most developers implement that using Azure API Management or container-based load balancers. A native Python approach that integrates with the OpenAI Python API Library is also possible. For integration instructions with this sample, please check:
```

File: docs/productionizing.md [hunk 2]
```diff
@@ -24,9 +24,10 @@ If the maximum TPM isn't enough for your expected load, you have a few options:
...
   * [Scale Azure OpenAI for Python with Azure API Management](https://learn.microsoft.com/azure/developer/python/get-started-app-chat-scaling-with-azure-api-management)
   * [Scale Azure OpenAI for Python chat using RAG with Azure Container Apps](https://learn.microsoft.com/azure/developer/python/get-started-app-chat-scaling-with-azure-container-apps)
+  * [Scale Azure OpenAI for Python with the Python openai-priority-loadbalancer](https://github.com/Azure-Samples/azure-search-openai-demo/pull/1626)
```
Comment (pamelafox) [inline docs/productionizing.md:30]:
```suggestion
  * [Pull request: Scale Azure OpenAI for Python with the Python openai-priority-loadbalancer](https://github.com/Azure-Samples/azure-search-openai-demo/pull/1626)   
```

**Review by pamelafox — APPROVED** [review]

**Comment by simonkurtz-MSFT** [comment]
#sign-off

**Comment by simonkurtz-MSFT** [comment]
@pamelafox, thank you! I also just updated the PR overview on #1626 to give more context to the PR.

**Comment by simonkurtz-MSFT** [comment]
> Ah our repo doesn't have the azure-docs-pr label magic. I'll merge! :)



> Ah our repo doesn't have the azure-docs-pr label magic. I'll merge! :)

Ha, you can tell what I was thinking with that then. =P

**Comment by pamelafox** [comment]
Oh great, thanks for updating the PR!

**Pull request merged**